src/main.rs

```diff
@@ -875,6 +885,33 @@ mod tests {
         .unwrap()
     }
 
+    // Produces an OnionMessage that can be used for tests. We need to manually write individual bytes because onion
+    // messages in LDK can only be created using read/write impls that deal with raw bytes (since some other fields
+    // are not public).
```

Comment:
Useful having any of these currently private fields be exposed in some way or some util, even at a higher level? Or is it really just for testing-sake.

Reply:
Should only be for testing! In the real world we'll be getting the bag-o-bytes from LND 🧅